lib/decidim/civicrm/version.rb

```diff
@@ -3,8 +3,8 @@
 module Decidim
   # This holds the decidim-civicrm version.
   module Civicrm
-    DECIDIM_VERSION = "0.28.3"
-    COMPAT_DECIDIM_VERSION = [">= 0.28.0", "< 0.29"].freeze
+    DECIDIM_VERSION = { github: "decidim/decidim", branch: "release/0.29-stable" }.freeze
```

Comment:
```suggestion
    DECIDIM_VERSION ="~> 0.29.2"
```
No need to edge branch here

Reply:
Done